src/Tasks/InvalidateAllSessionsTask.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace SilverStripe\SessionManager\Tasks;
+
+use SilverStripe\Dev\BuildTask;
```

Comment:
```suggestion
use SilverStripe\Control\HTTPRequest;
use SilverStripe\Dev\BuildTask;
```

Reply:
> Method 'SilverStripe\SessionManager\Tasks\InvalidateAllSessionsTask::run()' is not compatible with method 'SilverStripe\Dev\BuildTask::run()'

We can't add typehints here until they're also in framework.